backend/src/Matroos.Backend.Core/Classes/QueryFilter.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Matroos.Backend.Core.Classes;
+
+public class QueryFilter
+{
+    /// <summary>
+    /// Documents to be skip.
+    /// </summary>
+    public int Skip { get; set; } = 0;
```

Comment:
You *never, ever* use getters and setters of everything unless a user story requires you to do so. Will this change after construction? Is this an object value or an entity? How has this been designed according to domain driven design (or other methodology)?